.github/workflows/OperatorIntegrtionTest.yml

```diff
@@ -0,0 +1,163 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+name: .github/workflows/OperatorIntegrtionTest.yml
+env:
+  TERRAFORM_AWS_ASSUME_ROLE: ${{ secrets.TERRAFORM_AWS_ASSUME_ROLE }}
+  AWS_ASSUME_ROLE: ${{ secrets.TERRAFORM_AWS_ASSUME_ROLE }}
+  ECR_OPERATOR_STAGING_IMAGE: ${{ secrets.ECR_OPERATOR_STAGING_IMAGE }}
+
+on:
+  push:
+    branches:
+      - main*
+      - addOperatorIntgTest
+  workflow_dispatch:
+    inputs:
+      addon_name:
```

Comment:
I don't think the add-on inputs are needed here. We should push the latest build of the operator using the `BuildAndUploadStaging` job - https://github.com/aws/amazon-cloudwatch-agent-operator/blob/main/.github/workflows/build-and-upload-staging.yml

This will publish to `${{ secrets.ECR_OPERATOR_STAGING_IMAGE }}` which can be used for testing